.changes/unreleased/FEATURES-20231006-135721.yaml

```diff
@@ -0,0 +1,6 @@
+kind: FEATURES
```

Comment:
Change to `ENHANCEMENTS` because it's a new field

> ENHANCEMENTS: Smaller features added to the project such as a new field for an already existing API service or endpoint.
https://github.com/bastionzero/bastionzero-sdk-go/blob/master/CONTRIBUTING.md#changelog

Reply:
b574bc2d38a8307385ccf82081f6a1253b339e7c